cardano-cli/src/Cardano/CLI/EraBased/Query/Run.hs

```diff
@@ -805,7 +810,28 @@ runQueryLedgerStateCmd
 
             result <- easyRunQuery (queryDebugLedgerState sbe)
 
-            pure $ shelleyBasedEraConstraints sbe (writeLedgerState mOutFile) result
+            -- format
+            --       & ( id
```

Comment:
I take is this isn't ready for review?

Reply:
For some reason, Github opened an outdated version of the file for you.